app/build.gradle

```diff
@@ -134,92 +134,92 @@ ext {
 
 
 dependencies {
-    implementation "androidx.legacy:legacy-support-v4:$legacySupport"
-    debugImplementation "com.squareup.leakcanary:leakcanary-android:$leakCanary"
+    implementation AndroidX.legacy.supportV4
```

Comment:
the plugin provides read-to-use organized dependency notations for some libraries, this is why this has changed